content/topics/bootcamp/git-for-bootcamp/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: Git for Bootcamp
+ready: true
+---
+
+### Using git features with Spck
+
+
+### Basic Terminal Commands needed for Bootcamp
+|git command|Function|
+|---|---|
+|`git clone` *repo url*|clones the repo to your local machine|
```

Comment:
These kids are going to die

What is git? Why are we using it? What's a branch? 

Does it actually make sense to do full on git in the bootcamp? 

I guess there is a video coming, but right now it looks like we are setting people up for failure.

Maybe actually include the existing git excercises so they can get a feel for things? Can they do that stuff on a phone? I can make it that with certain Project cards the recruit can set the repo themselves

Reply:
There is a short video coming, this is also only the basic commands that they need to work in boot, the video is just an example of going throiugh the process of cloning etc.

There is a spck video aswell as it has a couple of built in features, but they will not be able to do the full git exercises if they are on android